packages/types/src/sign-client/client.ts

```diff
@@ -22,6 +22,7 @@ export declare namespace SignClientTypes {
   interface BaseEventArgs<T = unknown> {
     id: number;
     topic: string;
+    pairingTopic: string;
```

Comment:
Doesn't belong in the Base RPC event args, since this is a generic interface to represent any/all types of RPC events, by passing along the JSON-RPC `id`, the `topic` the event occurred on and some generic payload `T`.

I.e. this interface isn't aware of implementation details like pairings, sessions, etc

```suggestion

```